commerce.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+//go:generate go run github.com/swaggo/swag/cmd/swag init --generalInfo=commerce.go --output=docs/openapi
```

Comment:
This won't nicely integrate with customizations.
I think it might make more sense to integrate the swag-concept with a generic API module.
The API module can be found by swag (using dingo) and implement a route-module, to auto-generate as much as possible